### DISCUSSION TITLE
Show 4k and 7k ranks on lazer

### DISCUSSION THREAD

**Discussion by theredpro-sj** [ppy/osu#30787] [Ideas] [answered]
I'm not very confident in how I worded the title but the basic idea of it is that on the website when hovering global rank in the mania gamemode you also see the users 4k and 7k rank along with their peak rank like in other gamemodes (see first image). This behavior is not replicated on the lazer client as when hovering over global rank it only shows peak rank and doesn't show the 4k and 7k ranks (see second image). Basically I feel that the 4k and 7k breakdown should be shown when hovering over a users global rank like it is on the website.
<img width="513" alt="Screenshot 2024-11-19 at 6 12 39 PM" src="https://github.com/user-attachments/assets/6b9ae797-5b2b-449a-96f9-9373172e2d3b">

<img width="1470" alt="Screenshot 2024-11-19 at 6 11 39 PM" src="https://github.com/user-attachments/assets/3396f52d-4619-4864-b983-434d2d1dfe4e">

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/30827
